rnaseq/bam_to_fastq.cwl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Is this the same as https://github.com/genome/cancer-genomics-workflow/blob/master/unaligned_bam_to_bqsr/revert_to_fastq.cwl